### ISSUE TITLE
WorldGuard protection

### ISSUE THREAD

**Issue by SlimeDog** [ThizThizzyDizzy/tree-feller#4]
Spigot 1.14.4
TreeFeller 1.9.3

It looks like TreeFeller blocks in any WorldGuard region, including
```
regions:
  __global__:
...
```
which is the entire world. So if WorldGuard is installed, TreeFeller is effectively disabled. IMHO, TreeFeller should be blocked only in WorldGuard regions listed in config.yml, and only if
```
    flags:
      block-break: allow
```
or
```
    flags:
      build: allow
```
is not set (which is the default).

**Comment by ThizThizzyDizzy** [comment]
TreeFeller _should_ only block in regions the player doesn't have block-break access to...
The problem is this line:
`query.testState(location, player, Flags.BLOCK_BREAK)`
That returns `true` or `false` based on the flag, rather than `allow`, `deny`, and `none`
I assumed that `true` meant the player has access, and `false` meant they didn't, and it seemed to work when I tested it.

**Comment by ThizThizzyDizzy** [comment]
Alright, figured it out
It's only checking the `block-break` flag

**Comment by ThizThizzyDizzy** [comment]
And now I find I can use `queryState` instead of `testState` to get the allow/deny/etc.

**Comment by ThizThizzyDizzy** [comment]
Fixed in 1.9.4

**Comment by SlimeDog** [comment]
Looks OK to me, in 1.9.4. Thanks.
